### DISCUSSION TITLE
Efficient Team Collaboration During Absences

### DISCUSSION THREAD

**Discussion by 6senseHQ** [6sense-Technologies/6senseHQ-Discussions#3] [Team Work]
Notifying the team in advance of a leave/absence, especially for members who work directly together, is advised during daily planning meetings in the morning or at least one day beforehand. Proactive communication improves greater cooperation and supports seamless workflow management within the development team, as tasks are interdependent.

**It is not a part of any of our processes, we should follow this for team collaboration only.** 

Farjana

**Comment by ahsanaasim** [comment]
So, let's finalize a process to avoid conflicts & issues.

**For planned leaves:**
- **Mandatory:** Inform the project coordinator before submitting a leave request.
- **Mandatory:** Clear any dependencies with other team members to ensure no tasks are left incomplete due to your absence. Confirm with the project coordinator that all responsibilities have been properly managed.
- **Mandatory:** Notify the project coordinator and the entire team once the leave is approved.

**For emergency or sick leaves:**
- **Mandatory:** Inform the project coordinator & team lead before submitting a leave request.
- **Mandatory:** Notify the project coordinator once the leave is approved.
